From dec2b99fba2196a054001c9eca852272d9cf58a6 Mon Sep 17 00:00:00 2001
From: Ivan Vecera <ivecera@redhat.com>
Date: Wed, 19 Nov 2014 16:13:04 -0500
Subject: [ethernet] be2net: remove unnecessary blank lines after an open brace

Message-id: <1416413587-12091-17-git-send-email-ivecera@redhat.com>
Patchwork-id: 100336
O-Subject: [RHEL7.1 PATCH 16/19] be2net: remove unnecessary blank lines after an open brace
Bugzilla: 1165755
RH-Acked-by: Dean Nelson <dnelson@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>

BZ: #1165755

Upstream commit(s):
commit 619f2d1a8f2f94d31ae90480f8e39dc91d5a7360
Author: Kalesh AP <kalesh.purayil@emulex.com>
Date:   Fri Sep 19 15:46:59 2014 +0530

    be2net: remove unnecessary blank lines after an open brace

    This patch fixes checkpatch warnings about blank lines after an open brace '{'.

    Signed-off-by: Kalesh AP <kalesh.purayil@emulex.com>
    Signed-off-by: Sathya Perla <sathya.perla@emulex.com>
    Signed-off-by: David S. Miller <davem@davemloft.net>

Signed-off-by: Ivan Vecera <ivecera@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/emulex/benet/be_cmds.c b/drivers/net/ethernet/emulex/benet/be_cmds.c
index bc47d17..9ec09a8 100644
--- a/drivers/net/ethernet/emulex/benet/be_cmds.c
+++ b/drivers/net/ethernet/emulex/benet/be_cmds.c
@@ -209,7 +209,6 @@ static int be_mcc_compl_process(struct be_adapter *adapter,
 
  if (base_status != MCC_STATUS_SUCCESS &&
      !be_skip_err_log(opcode, base_status, addl_status)) {
-
   if (base_status == MCC_STATUS_UNAUTHORIZED_REQUEST) {
    dev_warn(&adapter->pdev->dev,
      "VF is not privileged to issue opcode %d-%d\n",
@@ -1522,7 +1521,6 @@ err:
 int lancer_cmd_get_pport_stats(struct be_adapter *adapter,
           struct be_dma_mem *nonemb_cmd)
 {
-
  struct be_mcc_wrb *wrb;
  struct lancer_cmd_req_pport_stats *req;
  int status = 0;
@@ -3038,7 +3036,6 @@ out:
 int be_cmd_get_active_mac(struct be_adapter *adapter, u32 curr_pmac_id,
      u8 *mac, u32 if_handle, bool active, u32 domain)
 {
-
  if (!active)
   be_cmd_get_mac_from_list(adapter, mac, &active, &curr_pmac_id,
       if_handle, domain);
diff --git a/drivers/net/ethernet/emulex/benet/be_main.c b/drivers/net/ethernet/emulex/benet/be_main.c
index 8b3a413..3707544 100644
--- a/drivers/net/ethernet/emulex/benet/be_main.c
+++ b/drivers/net/ethernet/emulex/benet/be_main.c
@@ -493,7 +493,6 @@ static void populate_be_v2_stats(struct be_adapter *adapter)
 
 static void populate_lancer_stats(struct be_adapter *adapter)
 {
-
  struct be_drv_stats *drvs = &adapter->drv_stats;
  struct lancer_pport_stats *pport_stats = pport_stats_from_cmd(adapter);
 
@@ -4737,7 +4736,6 @@ static void be_func_recovery_task(struct work_struct *work)
  be_detect_error(adapter);
 
  if (adapter->hw_error && lancer_chip(adapter)) {
-
   rtnl_lock();
   netif_device_detach(adapter->netdev);
   rtnl_unlock();
-- 
1.7.1